game/src/pregame.rs

```diff
@@ -497,15 +511,16 @@ impl Screensaver {
         }
     }
 
-    fn update(&mut self, rng: &mut XorShiftRng, ctx: &mut EventCtx, map: &Map) {
+    fn update(&mut self, rng: &mut XorShiftRng, ctx: &mut EventCtx, app: &mut App) {
         if ctx.input.nonblocking_is_update_event().is_some() {
             ctx.input.use_update_event();
             let dist_along = Duration::realtime_elapsed(self.started) * SPEED;
             if dist_along < self.line.length() {
+                app.primary.sim.tiny_step(&app.primary.map, &mut app.primary.sim_cb);
```

Comment:
This jumps 0.1 sim seconds roughly 30 times per real second (`UPDATE_FREQUENCY`), so it's equivalent to about 3x speed. But in practice the frame-rate could differ, so it'd be much safer to do `if let Some(dt) = ctx.input.nonblocking_is_update_event()` and then `                app.primary.sim.time_limited_step(
                    &app.primary.map,
                    multiplier * dt,
                    Duration::seconds(0.033),
                    &mut app.primary.sim_cb,
                );`, just like sandbox/speed does. Using `time_limited_step` prevents the title screen from becoming laggy if the sim gets slow. You could tune multiplier however you like; about 3x looks pretty good from the sample run I did.